PHP_CodeSniffer/Tokenizers/GraphQL.php

```diff
@@ -0,0 +1,258 @@
+<?php
+/**
+ * Copyright © Magento. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace PHP_CodeSniffer\Tokenizers;
```

Comment:
Should be under Magento namespace.

Reply:
Agreed, unfortunately PHP_CodeSniffer is not very fond of additional tokenizers and have therefore hardcoded the namespace under which tokenizers reside to `PHP_CodeSniffer\Tokenizers`: https://github.com/squizlabs/PHP_CodeSniffer/blob/master/src/Files/File.php#L562